1-exercises/F-for-of-loop/exercise.js

```diff
@@ -11,6 +11,15 @@ let tubeStations = [
     "Tottenham Court Road"
 ];
 
+for (i of tubeStations)
```

Comment:
```suggestion
for (const station of tubeStations)
```
I think it will be better to use a meaningful variables names, it's easy to read and understand ("i" generally refers to index which is not the case here)